pkg_security.Rmd

```diff
@@ -4,9 +4,17 @@
 This work-in-progress chapter includes [guidance about managing secrets in packages](#pkgsecrets) and [links for further reading](#furthersecreading).
 ```
 
-## Miscellaneous
+## Misc
```

Comment:
Just wondering why "Misc" and not the actual word "Miscellaneous"?

Reply:
Oops, changed! For the record the diff was first to remove the section then to re-add it.